bindings/wallet-jni/src/lib.rs

```diff
@@ -123,3 +121,101 @@ pub unsafe extern "system" fn Java_com_iohk_jormungandrwallet_Wallet_initialFund
     }
     settings as jlong
 }
+
+#[no_mangle]
+pub unsafe extern "system" fn Java_com_iohk_jormungandrwallet_Wallet_convert(
+    env: JNIEnv,
+    _: JClass,
+    wallet: jlong,
+    settings: jlong,
+) -> jlong {
+    let wallet_ptr = wallet as WalletPtr;
+    let settings_ptr = settings as SettingsPtr;
+    let mut conversion_out = null_mut();
+    let result = wallet_convert(
+        wallet_ptr,
+        settings_ptr,
+        (&mut conversion_out) as *mut ConversionPtr,
+    );
+
+    if let Some(error) = result.error() {
+        env.throw(error.to_string());
```

Comment:
I'm in a warnings hunting mode right now. Can we consume this result somehow?

Reply:
good catch, I actually let the warning on purpose to think about it later, and then I forgot. What @NicolasDP did for similar cases is this:
```rust 
let _ = env.throw(error.to_string());
```
so I guess I'll take the same route